README.md

```diff
@@ -24,7 +24,7 @@ The script simply try salts for the bytecode of that example.
 
 ## Compilation & Verification
 
-> :warning: The `solc` compiler version `0.5.11+commit.22be8592` must be present on the build machine. ([More info on installing solc][solc-install].)
+> :warning: The `solc` compiler version `0.7.3+commit.9bfce1f6` must be present on the build machine. ([More info on installing solc][solc-install].)
```

Comment:
why not 0.7.4?

Reply:
no reason at all, than svm didn't have at current time.